src/backend/common/forms.py

```diff
@@ -131,14 +164,23 @@ def to_dataclass(self) -> GroupActionsData:
             name=self.name.data,
             description=self.description.data,
             github_org=GitHubOrgData(
-                name=self.org.data,
-            )
+                github_id=self.org.data,
+            ),
+            actions_ids=[
+                int(action_id.strip()) for action_id in
+                self.actions_ids.data.split(',')
```

Comment:
```suggestion
                int(action_id.strip())
                for action_id in self.actions_ids.data.split(',')
```

This formatting makes it easier to see where is the result part and where is the loop itself